darwin/datatypes.py

```diff
@@ -43,6 +74,8 @@ class AnnotationFile:
     image_url: Optional[str] = None
     workview_url: Optional[str] = None
     seq: Optional[int] = None
+    is_video: Optional[bool] = None
```

Comment:
I think we can safely make this required, and default it to `false`
```suggestion
    is_video: bool = false
```
Let me know if you think that would be a wrong idea

Reply:
I agree, however, we should then imo reorder it so the optional fields are last

Reply:
Agree too. I changed it and reordered.